library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+        ** Periodic-respiration <nowiki> [Source: Beniczky ea 2017, Table 16] </nowiki>
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.(Source: Beniczky ea 2017, Table 16.)]</nowiki>
+        ** Tachypnea <nowiki> {suggestedTag=Frequency} [Numerical value for cycles / minute. (Source: Beniczky ea 2017, Table 16.)]</nowiki>
+            *** <nowiki># {takesValue, valueClass=numericClass} </nowiki>
+        ** Other-respiration-feature <nowiki> {requireChild} [Source: Beniczky ea 2017, Table 16] </nowiki>
             *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-        ** ECG-feature
-            *** ECG-sinus-rhythm <nowiki> {suggestedTag=Frequency} [Normal rhythm.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** ECG-arrhythmia
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** ECG-asystolia <nowiki>[Add duration (range in seconds) and comments in free text.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** ECG-bradycardia <nowiki> {suggestedTag=Frequency} </nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** ECG-extrasystole
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** ECG-ventricular-premature-depolarization <nowiki> {suggestedTag=Frequency} </nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** ECG-tachycardia <nowiki> {suggestedTag=Frequency} </nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Other-ECG-feature {requireChild}
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-    * EMG-channel-finding <nowiki> {suggestedTag=Finding-significance-to-recording} [EMG channel findings involve distinctive muscular activity patterns and characteristics observed in the Electromyography recording.]</nowiki>
-        ** EMG-muscle-side
-            *** EMG-left-muscle
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** EMG-right-muscle
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** EMG-bilateral-muscle
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+    * ECG-channel-finding <nowiki> {suggestedTag=Finding-significance-to-recording} [Findings in Electrocardiogram recordings. (Source: Beniczky ea 2017, Section 13, Table 16.)]</nowiki>
+        ** ECG-QT-period <nowiki> {requireChild}[Value and free text. (Source: Beniczky ea 2013, Table 9; Beniczky ea 2017, Table 16.)] </nowiki>
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
```

Comment:
I am not sure what is being asked for here. I think QT period is usually given in seconds, in which case it should be valueclass=numericClass, unitClass=timeUnits.

If it needs a text description, then maybe:

"A text description that includes an estimate of the period in seconds and a description of any abnormal features."

Reply:
This is confusing, as the paper Table 16 allows value + free text. The problem I have with that is that the value would then not be machine readable. If I assume that free text can events/comments can always be added, but should not be part of this tag.

Reply:
Fixed to only numericClass